subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/SequentialOutputMatcher.groovy

```diff
@@ -30,8 +30,8 @@ class SequentialOutputMatcher {
     private static final String NL = SystemProperties.instance.lineSeparator
 
     public void assertOutputMatches(String expected, String actual, boolean ignoreExtraLines) {
-        List actualLines = normaliseOutput(actual.readLines())
-        List expectedLines = expected.readLines()
+        List actualLines = normaliseOutput(actual.readLines()).findAll { !it.isEmpty() }
```

Comment:
When a warning is emitted, some extra empty lines are also appear on console:

```
> Task: a

> Task: test
WARNING:

> Task: b
> Task: c
```

This is to eliminate extra empty lines.